Gemfile.lock

```diff
@@ -429,6 +429,7 @@ GEM
 
 PLATFORMS
   arm64-darwin-21
+  arm64-darwin-22
```

Comment:
do we want to merge this in this PR?

Reply:
It would be be better separate, but I don't think it's harmful.